Calling/App.xaml.cs

```diff
@@ -12,6 +20,11 @@ namespace CallingQuickstart
     /// </summary>
     sealed partial class App : Application
     {
+        public string PNHChannelUri { get; set; }
```

Comment:
I wonder if you should just keep this a System.Uri type, so that you don't have to rely on string.IsNullOrEmpty() to validate it later, instead you could just do is null check.

Reply:
Changed to Uri